common.gypi

```diff
@@ -51,6 +53,37 @@
     ],
   },
 
+  'conditions': [
+    ['node_engine=="v8"', {
+      'target_defaults': {
+        'defines': [
+          'NODE_ENGINE="<(node_engine)"',
```

Comment:
This could probably be moved to a common "defines" array, since its duplicated 11 lines below.